### PR TITLE
Updated bower dependencies + Polymer 2.X hybrid mode

### PR DESCRIPTION
**Pull request by BorntraegerMarc**
Updated all bower dependencies
Enabled Polymer 2.X hybrid mode for all elements

### CODE REVIEW

**Comment by BorntraegerMarc** [comment]
Tests are running without problems and tested it in demo + own application. Seems to be working fine...

**Comment by BorntraegerMarc** [comment]
@ronnyroeller Would be cool if somebody could do at least a pre-release so I could use it in my projects and start testing in production...

File: paper-filter-dialog.html [hunk 1]
```diff
@@ -75,47 +75,51 @@
...
-				<iron-list id="valuesList" items="[[_selectedFilterValues]]" as="filterValue" class="list">
-					<template>
-						<div class="value-holder">
-							<paper-item class="value">
-								<paper-checkbox name="[[filterValue.id]]" checked="{{filterValue.selected}}"></paper-checkbox>
-								<div>
-									[[filterValue.name]]
-									<span class="nrResults" hidden="[[!filterValue.count]]">([[filterValue.count]])</span>
-								</div>
+    		<template is="dom-if" if="{{!_selectedFilter}}">
```
Comment (ronnyroeller) [inline paper-filter-dialog.html]: Could you please fix the indentation?

File: paper-filter-dialog.html [hunk 2]
```diff
@@ -75,47 +75,51 @@
...
+								</paper-item-body>
 							</paper-item>
-						</div>
-					</template>
-				</iron-list>
-			</div>
+						</template>
+					</iron-list>
+				</div>
+    		</template>
```
Comment (ronnyroeller) [inline paper-filter-dialog.html]: Could you please fix the indentation (also next line)?

File: paper-search-bar.html [hunk 3]
```diff
@@ -65,13 +68,19 @@
...
 			<iron-icon icon="[[icon]]" class="icon"></iron-icon>
-			<input is="iron-input" id="input" bind-value="{{query}}" placeholder="[[placeholder]]" class="flex"></input>
+			<iron-input id="input" bind-value="{{query}}" class="flex">
+				<input placeholder="[[placeholder]]">
+			</iron-input>
 		</div>
-		<paper-icon-button icon="clear" hidden$="[[!query]]" on-tap="_clear" class="icon"></paper-icon-button>
-		<div hidden$="[[hideFilterButton]]">
-			<paper-icon-button id="filter" icon="image:tune" on-tap="_filter" class="icon" disabled="[[disableFilterButton]]"></paper-icon-button>
+    	<template is="dom-if" if="{{query}}">
```
Comment (ronnyroeller) [inline paper-search-bar.html]: Could you please fix the indentation (also next 4 lines)?

File: paper-search-panel.html [hunk 4]
```diff
@@ -89,17 +90,23 @@
 			no-values-label="[[noValuesLabel]]">
 		</paper-filter-dialog>
 
-		<div hidden$="[[!_hasItems]]" class="items flex">
-			<content select=":not([fixed])"></content>
-		</div>
+    	<template is="dom-if" if="{{_hasItems}}">
```
Comment (ronnyroeller) [inline paper-search-panel.html]: Could you please fix the indentation (also next couple of lines)?

**Review by ronnyroeller — CHANGES_REQUESTED** [review]
Could you please fix the indentation? I hope to have pointed all where there are tabs missing/too many. Thanks!

**Comment by BorntraegerMarc** [comment]
@ronnyroeller Implemented the requested changes. Sorry, I had apparently some problems with my code editor... I'm usually very keen on indentation :)

**Comment by ronnyroeller** [comment]
No problem. Merged + release created.

**Pull request merged**